azure-pipelines.yml

```diff
@@ -12,18 +12,26 @@ variables:
   tests: './src/Minsk.Tests/Minsk.Tests.csproj'
 
 steps:
-- task: UseDotNet@2
-  displayName: Use .NET 5.0 Preview
-  inputs:
-    version: 5.0.x
-    packageType: sdk
-    includePreviewVersions: true
-# We now still need .NET Core 3.1 Runtime for CI tests
-- task: UseDotNet@2
-  displayName: Install .NET Core 3.1 Runtime
-  inputs:
-    version: 3.1.x
-    packageType: runtime
+# TODO: Remove the following crazy hack once a properly listed Preview SDK for Source Generators feature becomes available.
+- script: |
+    curl -o "$AGENT_TEMPDIRECTORY/dotnet-install.sh" -sSL https://dot.net/v1/dotnet-install.sh
+    bash "$AGENT_TEMPDIRECTORY/dotnet-install.sh" --version 3.1.300-preview-015135 --install-dir "$AGENT_TOOLSDIRECTORY/dotnet" --verbose
+    echo "##vso[task.setvariable variable=Dotnet.Root;]$AGENT_TOOLSDIRECTORY/dotnet"
+    echo "##vso[task.prependpath]$AGENT_TOOLSDIRECTORY/dotnet"
+  displayName: Use .NET Core 3.1.3 Preview SDK
+# TODO: Uncomment lines below once Preview/Release SDK for Source Generators feature becomes available.
```

Comment:
We have `git revert` so, it's best that we don't include commented out section in the commit.

Reply:
Yeah, but in this case I prefer the commented version because it provides additional context.